pkg/applicationserver/observability.go

```diff
@@ -45,7 +45,7 @@ var (
 	evtReceiveDataDown      = events.Define("as.down.data.receive", "receive downlink data message")
 	evtDropDataDown         = events.Define("as.down.data.drop", "drop downlink data message")
 	evtForwardDataDown      = events.Define("as.down.data.forward", "forward downlink data message")
-	evtLostQueueDataDown    = events.Define("as.down.data.queue.lost", "lost downlink data queue")
+	evtLostQueueDataDown    = events.Define("as.down.data.queue.lost", "lose downlink data queue")
```

Comment:
Not sure about this one. @johanstokking ?

Reply:
Yeah good catch @bafonins. The event names are imperative